app/src/newdocs/content/tutorials.mdx

```diff
@@ -1,12 +1,239 @@
 import ScrollableAnchor from 'react-scrollable-anchor'
+import { Link } from 'react-router-dom'
 
-import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
+import Warning from '$docs/components/Warning'
+import CodeSnippet from '$shared/components/CodeSnippet'
+import links from '$shared/../links'
+import { formatPath } from '$shared/utils/url'
 
-## Tutorials
+import CreateStream from './images/tutorials/createStream.png'
+import StreamSetup from './images/tutorials/streamSetup.png'
+import GetYourAPIKey from './images/tutorials/getYourAPIKey.png'
+import SettingsAPICredentials from './images/tutorials/settingsAPICredentials.png'
+import Postman from './images/tutorials/postman.png'
+import Example2b from './images/tutorials/example2b.png'
+import Example3b from './images/tutorials/example3b.png'
 
-<ScrollableAnchor id="placeholder"><div>
+import { ClientSub, Example1, Example2a, Example3a, PythonRequests } from './code/tutorials.js'
 
-## First tutorial placeholder
---content--
+import docsStyles from '$docs/components/DocsLayout/docsLayout.pcss'
+
+# First Tutorial
+
+The aim of this tutorial is to provide an overview of Streamr basic data flows:
+
+- What a data stream is within the context of Streamr
+- How to create a simple pub/sub demo
+- Common data integration patterns
+
+By the end of this tutorial, you will learn how to create a stream and publish data to it, and how you can obtain/visualize data by subscribing to a stream. In the near future, we will be providing additional tutorials that will go further in depth about specific topics or use-cases.
+
+## About Streams
```

Comment:
I think this can be left out and moved to the streams section (unless tutorials should be self contained).

Reply:
I think during our last call we mentioned that some repetition would be fine as people might not read the Docs in the sequential order. But I'm open to remove the intro section and link to the Streams section inside the tutorial for a more in-depth reading.